Samples.xcodeproj/project.pbxproj

```diff
@@ -615,6 +620,9 @@
 			dstPath = "";
 			dstSubfolderSpec = 7;
 			files = (
+				1C293D012DCA7C99000B0822 /* ApplyBlendRendererToHillshadeView.swift in Copy Source Code Files */,
+				1C293D032DCA7C99000B0822 /* ApplyBlendRendererToHillshadeView.SettingsView.swift in Copy Source Code Files */,
+				00FA4E592DBC1786008A34CF /* AddRasterFromGeopackageView.swift in Copy Source Code Files */,
```

Comment:
This line shouldn't be here. Remember to always have Xcode re-save the project file (eg. by renaming or rearranging a file and then undoing) after making manual modifications to it. When you do that for this project, this line will go away.